<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>iA4DMainWin</class>
 <widget class="QMainWindow" name="iA4DMainWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>406</width>
    <height>723</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="layoutStage">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="widget_2" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>20</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>20</height>
               </size>
              </property>
              <property name="text">
               <string>Force (Newtons)</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_3" native="true">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>200</height>
            </size>
           </property>
           <property name="sizeIncrement">
            <size>
             <width>0</width>
             <height>200</height>
            </size>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Preview</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_4" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Assets</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>64</width>
     <height>32</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSave"/>
   <addaction name="actionDefectFinder"/>
   <addaction name="actionVisualization"/>
   <addaction name="actionAdd"/>
  </widget>
  <action name="actionDefectFinder">
   <property name="icon">
    <iconset resource="../../iAnalyse.qrc">
     <normaloff>:/images/4DCT_Find.png</normaloff>:/images/4DCT_Find.png</iconset>
   </property>
   <property name="text">
    <string>Defect finder</string>
   </property>
  </action>
  <action name="actionVisualization">
   <property name="icon">
    <iconset resource="../../iAnalyse.qrc">
     <normaloff>:/images/4DCT_Visualization.png</normaloff>:/images/4DCT_Visualization.png</iconset>
   </property>
   <property name="text">
    <string>Visualization</string>
   </property>
   <property name="toolTip">
    <string>Visualization</string>
   </property>
  </action>
  <action name="actionAdd">
   <property name="icon">
    <iconset resource="../../iAnalyse.qrc">
     <normaloff>:/images/4DCT_Add.png</normaloff>:/images/4DCT_Add.png</iconset>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="../../iAnalyse.qrc">
     <normaloff>:/images/4DCT_Save.png</normaloff>:/images/4DCT_Save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../iAnalyse.qrc"/>
 </resources>
 <connections/>
</ui>
